lib/runCode.js

```diff
@@ -248,7 +248,6 @@ module.exports = function (opts, cb) {
     if (results.exceptionError) {
       delete results.gasRefund
       delete results.selfdestruct
-      self.stateManager.touched = []
```

Comment:
Hi Matthew, I am just doing a second review on this before we put this out on npm as a release. Just for my understanding: why is this here not needed anymore?

Reply:
The only way `exceptionError` can be truthy is if `err` is truthy. In this case `stateManager.revertContracts()` will have been called on ln 233. That method clears `_touched`

Reply:
Ok, thanks.